src/Workspaces/Core/Desktop/Workspace/MSBuild/VisualBasic/VisualBasicProjectFileLoader.cs

```diff
@@ -880,7 +880,7 @@ public bool SetLanguageVersion(string languageVersion)
                 {
                     if (!string.IsNullOrWhiteSpace(languageVersion))
                     {
-                        _commandLineArgs.Add("/languageversion:" + languageVersion);
+                        _commandLineArgs.Add("/langversion:" + languageVersion);
```

Comment:
Did this never work, or did we take a breaking change in the name of the argument?